docs/deploy-hyperlane.mdx

```diff
@@ -21,11 +21,11 @@ This guide has been specifically made to demonstrate how to connect a chain that
 
 There are five steps in this guide:
 
-1. [<b>Set up keys</b>](#1-set-up-keys) that you will use to deploy contracts and run validators and relayers.
+1. [<b>Set up keys</b>](#1-set-up-keys) that you will use to deploy contracts and run validators and relayer.
 2. [<b>Deploy contracts</b>](#2-deploy-contracts) to the local chain and to every remote chain with which the local chain will be able to send and receive messages.
-3. [<b>Run validators and relayers </b>](#3-run-validators-relayers) using [Kurtosis](https://cloud.kurtosis.com). Validators provide the signatures needed for the Interchain Security Modules you deployed in step 2 and relayers send and receive messages between chains you deployed contracts to in step 2.
-4. [<b>Send a test message</b>](#4-send-test-messages) to confirm that your relayer is able to deliver messages to and from each pair of chains.
-5. [<b>Deploy a Warp Route</b>](#5-optional-deploy-a-warp-route) to send token value, not just messages, across chains.
+3. [<b>Run validators and relayer </b>](#3-run-validators-relayer) using [Kurtosis](https://cloud.kurtosis.com). Validators provide the signatures needed for the Interchain Security Modules you deployed in step 2. The relayer will send and receive messages between chains you deployed contracts to.
```

Comment:
nit: i know this wasn't your change, but `The relayer will send and receive messages` feels slightly inaccurate (I'd say that users / smart contracts are the ones sending and receiving messages), I wonder if it should say `The relayer will deliver messages`

Reply:
deliver is more in-line with the phrasing used in other places